src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -91,6 +91,11 @@ export const serveForm: ControllerHandler<
     // Check the rollout value first, if it's 0, react is DISABLED
     // And we ignore cookies entirely!
     showReact = false
+    // Delete existing cookies to prevent infinite redirection
+    if (req.cookies) {
+      res.cookie(config.reactMigration.respondentCookieName, '', { maxAge: 0 })
+      res.cookie(config.reactMigration.adminCookieName, '', { maxAge: 0 })
```

Comment:
Use the [`res.clearCookie()`](https://expressjs.com/en/api.html#res.clearCookie) method instead. Same comment for call below.